examples/fundamentals__chrome-remote-debugging/cypress/plugins/index.js

```diff
@@ -0,0 +1,58 @@
+const CDP = require('chrome-remote-interface');
+
+function ensureRdpPort(args) {
+    const existing = args.find(arg => arg.slice(0, 23) === '--remote-debugging-port')
+
+    if (existing) {
+        return Number(existing.split('=')[1])
+    }
+
+    const port = 40000 + Math.round(Math.random() * 25000)
+    args.push(`--remote-debugging-port=${port}`)
+    return port
+}
+
+let port = 0;
+let client = null;
+
+module.exports = (on, config) => {
+    on('before:browser:launch', (browser, args) => {
+        port = ensureRdpPort(args);
```

Comment:
Change this to `ensureRdpPort(args.args);` to fix

Reply:
thanks for your hint. I will create a PR

Reply:
I tried it with cypress 4.4.0 and it works. The arguments are in args. What version do you have?

Reply:
I'm using the newest version. For your code it should throw a deprecation notice at least. Change was done in v4.0.0: https://docs.cypress.io/api/plugins/browser-launch-api.html#History

Reply:
I see. This was already implemented in https://github.com/cypress-io/cypress-example-recipes/commit/167e09829dde00aa1c39496f1e1f19babbc22157#diff-eef28f6bf842a3687a186b3cce136fb6

Reply:
Good hint, thanks! I found this PR from extensive googling & searching on Github, that's why I copied your code and wanted to note down the fix. If the real recipe is fixed, then all is well :)